packages/components/addon/components/hds/side-nav/section/item.hbs

```diff
@@ -0,0 +1,27 @@
+<li class="hds-side-nav__list-item">
+  <Hds::Interactive
+    @href={{@href}}
+    @route={{@route}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @icon={{@icon}}
+    @isRouteExternal={{@isRouteExternal}}
+    @current-when={{@current-when}}
+    class="hds-side-nav__link hds-typography-body-200 hds-font-weight-medium"
+    ...attributes
+  >
+    {{#if @icon}}
+      <FlightIcon class="hds-side-nav__list-icon" @name={{@icon}} @stretched={{true}} />
+    {{/if}}
+
+    {{@label}}
+
+    {{#if @badge}}
+      <Hds::Badge @text={{@badge}} @type="inverted" @size="small" />
```

Comment:
the badge need to support multiple color variants (you can see it in the Cloud UI codebase)

Reply:
What I tried first was just using a yield in the component which would allow users to add any content including badges inside. Would that be better? If so I can change it back. (I changed the API to add the arguments instead because I thought it would make it easier to use but I wasn't sure what was best.)

Reply:
I will redo this to support different variants.